meetings.py

```diff
@@ -43,8 +49,15 @@ def get_meetings_from(date, type, key, secret):
             raise ZoomApiException(response['error'])
 
         for meeting in response['meetings']:
-            meeting['type'] = response['type']
-            meetings.append(meeting)
+            document = create_meeting_document(meeting, response['type'])
+            meetings.append(document)
+
+            es.index(
+                index="meetings",
+                doc_type="meeting",
+                id=meeting['uuid'],
```

Comment:
remind me the diff between `uuid` and `id`? We need to differentiate between the abstract "meeting" and the occurrences of the meetings. I think it's the latter that we want to index.